src/main/java/step3/lotto/domain/lotto/MatchResult.java

```diff
@@ -0,0 +1,45 @@
+package step3.lotto.domain.lotto;
+
+/**
+ * @author : choi-ys
+ * @date : 2022/05/17 1:42 오후
+ */
+public enum MatchResult {
+    FIRST_PLACE(6, 2000000000),
```

Comment:
```suggestion
    FIRST_PLACE(6, 2_000_000_000),
```

1000 이 넘어가는 단위는 '_' 를 통해 가독성을 높혀볼 수 있어요 :)

Reply:
아하! 몰랐던 표현법을 또 알아가네요!
다음 미션 부터 '_'를 이용하여 수에 대한 표현을 가독성있게 작성해보겠습니다! 👍